lizrd/scripts/sync_with_remote.py

```diff
@@ -0,0 +1,65 @@
+import os
+from fabric import Connection
+from argparse import ArgumentParser
+from git import Repo
+from lizrd.support.code_versioning_support import find_git_root
+
+
+def go_to_sparsity():
+    git_root = find_git_root()
+    repo = Repo(git_root)
+    assert repo.remotes.origin.url in [
+        "git@github.com:llm-random/llm-random.git",
+    ], "You're not in the right repo! Move to the llm-random folder, and make sure your origin is the llm-random repo. Aborting..."
+    os.chdir(git_root)
+
+
+def rsync_to_remote(host, local_dir):
+    try:
+        with Connection(host) as c:
+            base_dir = get_base_directory(c)
+            rsync_command = (
+                f"rsync -rlp -e ssh {local_dir} {c.user}@{c.host}:{base_dir}"
```

Comment:
Jeszcze poproszę: `-z`: kompresja, `-h`: human readable, `--progress`: samo się tłumaczy, pomoże uniknąć problemów jakby ktoś miał jakieś duże pliki w `llm-random` (chociaż to human readable i progress tylko jeśli ten output się pokazuje u wykonującego).

Reply:
1. -z  dodam
2. nie piszę outputu; output printowy jest używany jako input do skryptu bashowego (brzydkie, wiem)

Reply:
syncuję specjalnie wyłącznie research i lizrd, bo to jest chudziutkie oraz wystarcza do puszczania ekspów